src/virtualenv/discovery/windows/__init__.py

```diff
@@ -13,6 +13,8 @@ def propose_interpreters(spec, cache_dir):
     # see if PEP-514 entries are good
     for name, major, minor, arch, exe, _ in discover_pythons():
         # pre-filter
+        if name in ("PythonCore",):
```

Comment:
the second argument is the python implementation, we know per PEP-514 that PythonCore is used by CPython